queries/gdscript/highlights.scm

```diff
@@ -1,18 +1,24 @@
 ;; Basic
+(ERROR) @error
+
 (identifier) @variable
 (name) @variable
 (type) @type
-(comment) @comment
-(get_node) @string
+(comment) @comment @spell
+(get_node) @string @symbol    ; fallback to string
+(string_name) @string @symbol
+(node_path) @string @symbol
```

Comment:
Why are these `@symbol`? For node_path, if it's a location/uri-like string, maybe `@text.uri`?

Reply:
Ah yes, `@text.uri` fits better than `@symbol`.

`get_node` is sugar for getting a ref to a node in godot's "scene tree" (`$get/this/node`). It's symbol like I would say. What would you suggest?

And now that I see it `string_name` should actually just be `@string`.

Reply:
Great suggestion, highlight support for `@text.uri` looks really good on both `get_node` and `node_path`.

Reply:
No, no tool should really be using that to determine links/paths to open, and I don't know of any. I'll leave this one up to you though, since the `$` beforehand is a bit funky but I'm not familiar with godot at all.

Reply:
Okay, I'll keep the change to `@text.uri`.